src/views/Pools/components/PoolCard/CardActions/StakeActions.tsx

```diff
@@ -32,17 +32,16 @@ const StakeAction: React.FC<StakeActionsProps> = ({
   isStaked,
   isLoading = false,
 }) => {
-  const { stakingToken, earningToken, stakingLimit, isFinished } = pool
+  const { stakingToken, earningToken, stakingLimit, isFinished, userData } = pool
   const { t } = useTranslation()
-  const convertedLimit = getDecimalAmount(new BigNumber(stakingLimit), earningToken.decimals)
+  const convertedLimit = getDecimalAmount(stakingLimit, earningToken.decimals)
```

Comment:
Here is an example of what I mentioned earlier. `stakingLimit` was already typed as a BigNumber so we shouldn't need to cast it again `new BigNumber(stakingLimit)`.